src/ServicePulse.Host/vue/src/components/monitoring/D3LargeGraph.stories.js

```diff
@@ -0,0 +1,57 @@
+import D3LargeGraph from "./D3LargeGraph.vue";
+/*import { expect } from "@storybook/jest";*/
+export default {
+    component: D3LargeGraph,
+    title: "Monitoring/EndpointDetails/D3LargeGraph",
+    tags: ["autodocs"],
+    //👇 Our events will be mapped in Storybook UI
+    argTypes: {},
+};
+
+//stories
+
+export const Default = {
+
+    minimumyaxis: 10,
+    avglabelcolor: "#EA7E00",
+    isdurationgraph: false,
+    metricsuffix: "MSGS",
+    csclass: "large-graph pull-left",
+    args: {
+        firstdataseries: {
+            average: 0.0,
+            points: [0.0, 0.0, 0.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 0.0, 2.0, 1.0, 1.0, 1.0, 1.0, 1.0, 2.0, 1.0, 1.0, 1.0, 1.0, 0.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0],
+        },
+    plotwidth: 750,
+    plotheight: 200,
+    firstseriescolor: "#EA7E00",
+    firstseriesfillcolor: "#EADDCE",
+    secondseriescolor: "",
+    secondseriesfillcolor: "",
+  },
+};
+
+export const ThroughputRetries = {
```

Comment:
I see the story data passes two different data series; however, the story doesn't showcase the distinct two different colors nor the average line. This is how it looks when I run it:

<img width="1057" alt="image" src="https://github.com/Particular/ServicePulse/assets/545695/614cad17-6df5-4ee8-98a8-96e9c90d9930">

But I was expecting to see something like this. This screenshot uses different data, colors, and scale units, but you get the idea. It showcases two data series with different colors and shows the average line. 

<img width="424" alt="image" src="https://github.com/Particular/ServicePulse/assets/545695/06766013-be9f-40d4-ad19-5179d94f7d28">

Reply:
The datapoints for both the first and second series are the same and hence the graphs overlapped. I have modified that second data series so that the 2 graphs are seen more distinctly.